### PR TITLE
Consider the biggest side of the image as the width for the resize

### PR DESCRIPTION
**Pull request by danmcgoo**
I was thinking that since the image orientation (portrait vs landscape), especially for snes, we should consider a way to have images which have the same area, regardless of their orientation.

This is just a suggestion. Since, it is my first attempt to write Go code, so this is a quick hack.
So, there is room for improvement.
At least, you get the idea..

### CODE REVIEW

**Comment by sselph** [comment]
I apologize, I haven't had a chance to look at this. I'll try to get to this tomorrow.

File: rom/rom.go [hunk 1]
```diff
@@ -328,8 +328,16 @@ func getImage(url string, p string, w uint) error {
 	if err != nil {
 		return err
 	}
-	if w > 0 && uint(img.Bounds().Dx()) > w {
-		img = resize.Resize(w, 0, img, resize.Bilinear)
+	if w > 0 {
+		if uint(img.Bounds().Dx()) >= uint(img.Bounds().Dy()) {
```
Comment (sselph) [inline rom/rom.go]: You don't need to convert both Dx() and Dy() to a uint comparison
`img.Bounds().Dx() >= img.Bounds().Dy()`
would work

**Comment by sselph** [comment]
Sorry for taking so long to get to this. I like the idea but the documentation and flag name are going to be a little confusing right it is `-max_width` "The max width of images. Larger images will be resized." Renaming it will likely break RetroPie's script and possibly others. We could possibly add a `-max_height` that when set to 0, which can be the default, it is set to the same as max_width?

File: rom/rom.go (hunk 1 above)
Comment (danmcgoo) [inline rom/rom.go]: OK. Didn't know.
Actually I added that line when I had issues to compile locally.
Because I thought my changes did not worked.

**Comment by danmcgoo** [comment]
As I said this is just a suggestion, you can reject the pull request and create your own solution.

I agree that maybe we should introduce another option, and maybe we could deprecate the `-max_width` option.
Actually with some thought on side too, I think it would be better to have a `-max_resolution` option, and apply the changes on the shortest side of the image.
It may be more understandable nowadays with HD resolution terminology (720, 1080, etc.)

**Comment by sselph** [comment]
I added a max_height flag and sent a PR to add it to RetroPie's menu.

**Pull request closed without merging**